app/usecases/uniNotifier.js

```diff
@@ -1,5 +1,19 @@
+const vvs = require('../services/vvs')();
+
 module.exports = function() {
   this.onUpdate = (ctx)=>{
+    if (ctx.update.message.text == 'vvs') {
```

Comment:
Should probably bee `===`

Reply:
shouldn't harm to use == (allows different types that have the same valu (3=="3") is true but in this case this shouldn't make any difference, or does it?